scripts/events_module/generate_events.py

```diff
@@ -131,13 +126,17 @@ def generate_short_events(event_triggered, biome):
                         print(
                             f"WARNING: some events resources which are used in generate_events have no 'event_text'."
                         )
+
+                    if frequency != event_frequency:
+                        continue
+
                     event = ShortEvent(
                         event_id=event["event_id"] if "event_id" in event else "",
                         location=event["location"] if "location" in event else ["any"],
                         season=event["season"] if "season" in event else ["any"],
                         sub_type=event["sub_type"] if "sub_type" in event else [],
                         tags=event["tags"] if "tags" in event else [],
-                        weight=event["weight"] if "weight" in event else 20,
+                        frequency=event["frequency"] if "frequency" in event else 4,
```

Comment:
```suggestion
                        frequency=event_frequency,
```
we've already determined this on line `119`, should just use `event_frequency`

also, if we've already discarded non-matching frequencies, why do we need to add this to the ShortEvent item?

Reply:
Ah, good point